### PR TITLE
Diversity Page 

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

**Comment by ViralBShah** [comment]
I would just make it `julialang.org/diversity`

File: content/community/diversity.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+> Although we have phrased the formal diversity statement generically to make it all-inclusive, we recognize that there are specific identities that are impacted by systemic discrimination and marginalization. We welcome all people to participate in the NumFOCUS community regardless of their identity or background.
+
+We believe that having diversity and inclusion in our communtiy is the only way we can have a strong and vibrant community, we have and will countinue to make significant investments in making sure that we relentlessly push towards that goal. 
+
+Below you will find a breakdown of our past and current diversity and inclusion initatives. If you have any ideas, feedback, or the like, feel free to send and email to diveristy@julialang.org or drop by the #diversity channel on Slack.
+
+ When                   | What                        | Link                          
+ -----------------------| --------------------------- | ---------------------------  
+ January 2020 | JuliaAcademy Overhall | https://juliaacademy.com
```
Comment (ViralBShah) [inline content/community/diversity.md]: overhaul spelling

**Comment by ViralBShah** [comment]
cc @xorjane

File: content/diversity/_index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> We have created this diversity statement because we believe that a diverse community is stronger, more vibrant, and produces better software and better science. A diverse community where people treat each other with respect has more potential contributors, more sources for ideas, and fewer shared assumptions that might hinder development or research.
+
+> Although we have phrased the formal diversity statement generically to make it all-inclusive, we recognize that there are specific identities that are impacted by systemic discrimination and marginalization. We welcome all people to participate in the NumFOCUS community regardless of their identity or background.
+
+We believe that having diversity and inclusion in our communtiy is the only way we can have a strong and vibrant community, we have and will countinue to make significant investments in making sure that we relentlessly push towards that goal. 
+
+Below you will find a breakdown of our past and current diversity and inclusion initatives. If you have any ideas, feedback, or the like, feel free to send and email to diveristy@julialang.org or drop by the [#diversity channel on Slack](https://slackinvite.julialang.org).
```
Comment (logankilpatrick) [inline content/diversity/_index.md]: @ViralBShah can we make a diversity@julialang.org email or shall we use an existing email for this?

**Pull request merged**